stack.yaml

```diff
@@ -0,0 +1,61 @@
+resolver: lts-5.13
```

Comment:
Can you explain what's going on with this `stack.yaml`?  Why all the very specific commits and so on?  Is this something we need/want in the repo itself?